### PR TITLE
chore: update to v13

### PR DESCRIPTION
**Pull request by yaruwangway**
## Description

Closes: #XXXX

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

* [ ] Included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
* [ ] Added `!` to the type prefix if API, client, or state breaking change (i.e., requires minor or major version bump)
* [ ] Targeted the correct branch (see [PR Targeting](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
* [ ] Provided a link to the relevant issue or specification
* [ ] Followed the guidelines for [building SDK modules](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/building-modules)
* [ ] Included the necessary unit and integration [tests](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#testing)
* [ ] Added a changelog entry in `.changelog` (for details, see [contributing guidelines](../../CONTRIBUTING.md#changelog))
* [ ] Included comments for [documenting Go code](https://blog.golang.org/godoc)
* [ ] Updated the relevant documentation or specification
* [ ] Reviewed "Files changed" and left comments if necessary 
* [ ] Confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

* [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
* [ ] confirmed `!` in the type prefix if API or client breaking change
* [ ] confirmed all author checklist items have been addressed 
* [ ] reviewed state machine logic
* [ ] reviewed API design and naming
* [ ] reviewed documentation is accurate
* [ ] reviewed tests and test coverage

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -149,7 +149,7 @@ jobs:
       # the old gaiad binary version is hardcoded, need to be updated each major release.
       - name: Install Old Gaiad
         run: |
-          git checkout v11.0.0 
+          git checkout v12.0.0-rc0
```
Comment (mpoke) [inline .github/workflows/test.yml]: Shouldn't this be v12.0.0?

**Review by p-offtermatt — APPROVED** [review]
Thanks for the work!

All changes outside of `app/upgrades/v13/*`, `contrib/scripts/upgrade_test_scripts/run_gaia.sh`are just replacing v12 with v13, and nothing looked problematic except perhaps Marius' comment. I grepd in the repo for v12 and all places where it's left look correct to me, so nothing was missed as far as I can tell.

File: .github/workflows/test.yml (hunk 1 above)
Comment (yaruwangway) [reply .github/workflows/test.yml]: v12.0.0 is not released yet

Comment (yaruwangway) [reply .github/workflows/test.yml]: can wait for v12.0.0 release, and then change here then merge PR.

Comment (mpoke) [reply .github/workflows/test.yml]: @yaruwangway https://github.com/cosmos/gaia/releases/tag/v12.0.0

**Review by mpoke — APPROVED** [review]

**Review by MSalopek — APPROVED** [review]
Making a note to myself that this can be done a bit differently.

The changes are related to this issue and PR:
* https://github.com/cosmos/gaia/issues/2255
* https://github.com/cosmos/gaia/pull/2692

**Pull request merged**